<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="team">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="add_team">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_schedule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Schedule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="store_schedule">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Store Schedule</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="home_team">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_matches">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch the sequence of matches</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switch_team_order">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>switch home/guest team for match</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="number_of_match_to_switch">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>340</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="guest_team">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="tournamen_start_at">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>118</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>match duration(minutes)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>break between matches</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="match_duration">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="break_between_matches">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="team_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="show_schedule">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
